### ISSUE TITLE
warn: Unable to uglify files

### ISSUE THREAD

**Issue by ziszo** [Polymer/polymer-cli#307]
Node: 4.2.6
Polymer-CLI: 0.12.0

How can I remove the "Unable to uglify file ..." warning?
What's the exact solution for this?

![image](https://cloud.githubusercontent.com/assets/4342864/16869236/7252c3a0-4aae-11e6-8a3a-9067e245c3f7.png)

**Comment by oneezy** [comment]
Same issue on this end...

-------------
**OS:** Windows 10 (64-bit)
**Git SCM:** 2.7.0 (MINGW64)
**Node:** 6.3.0
**Polymer-CLI:** 0.12.0

-------------

```
Oneeezy@Oneezy MINGW64 ~/Desktop/www_/_clients/diesel/wwwSFI (default)
$ polymer build

info:    Building application...
info:    Generating build/unbundled...
info:    Generating build/bundled...

warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\demo\index.html_script_0.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_0.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_1.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_2.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_3.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_4.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\web-component-tester\data\index.html_script_0.js
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\web-component-tester\data\index.html_script_2.js

info:    Generating service workers...
info:    Build complete!

Oneeezy@Oneezy MINGW64 ~/Desktop/www_/_clients/diesel/wwwSFI (default)

```


Any ideas?

**Comment by oneezy** [comment]
@ziszo , any luck??

**Comment by ziszo** [comment]
got no solution yet.

**Comment by isaldarriaga** [comment]
A simple **polymer build --verbose** should work to show you the syntax error reported by uglify

**Comment by oneezy** [comment]
Ok... using `polymer build --verbose`,

I'm getting a few syntax errors:

```
'SyntaxError: Unexpected character \'#\'',
'SyntaxError: Unexpected character \'`\''
```

**Full output:**
```
Oneeezy@Oneezy MINGW64 ~/Desktop/www_/_clients/diesel/wwwSFI (firebase)
$ polymer build --verbose
debug:   got args:
{ args: [ 'build', '--verbose' ] }
debug:   got default config from file:
{ config:
   { entrypoint: 'index.html',
     shell: 'src/shop-app.html',
     fragments:
      [ 'src/shop-list.html',
        'src/shop-detail.html',
        'src/shop-cart.html',
        'src/shop-checkout.html',
        'src/app-resources.html' ] } }
debug:   adding command build
debug:   adding command help
debug:   adding command init
debug:   adding command lint
debug:   adding command serve
debug:   adding command test
debug:   running...
debug:   command 'build' found, parsing command args:
{ args: [ '--verbose' ] }
debug:   command options parsed from args:
{ 'sw-precache-config': 'sw-precache-config.js', verbose: true }
debug:   final project configuration generated:
{ root: 'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI',
  entrypoint: 'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\index.html',
  shell: 'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-app.html',
  fragments:
   [ 'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-list.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-detail.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-cart.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-checkout.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\app-resources.html' ],
  inputs:
   [ 'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\index.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-app.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-list.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-detail.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-cart.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\shop-checkout.html',
     'C:\\Users\\Oneeezy\\Desktop\\www_\\_clients\\diesel\\wwwSFI\\src\\app-resources.html' ] }
debug:   Running command...
debug:   building with options
{ sources: undefined,
  swPrecacheConfig: 'sw-precache-config.js',
  insertDependencyLinks: undefined,
  html: {},
  css: {},
  js: {} }
debug:   building via standard build()...
debug:   shell: C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-app.html
debug:   entrypoint: C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\index.html
debug:   dependencies: C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\**\*
debug:   sources: C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\index.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-app.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-list.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-detail.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-cart.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-checkout.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\app-resources.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\**\*,!build,!build/**/*,!bower_components,!bower_components/**/*,!node_modules,!node_modules/**/*
debug:   fragments: C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-list.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-detail.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-cart.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\shop-checkout.html,C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\src\app-resources.html
debug:   no gulpfile found (searched up from C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI)
info:    Building application...
debug:   Reading source files...
debug:   Reading dependencies...
debug:   Analyzing build dependencies...
info:    Generating build/unbundled...
info:    Generating build/bundled...
error:   Uncaught exception: TypeError: Cannot read property 'previous' of null
error:   TypeError: Cannot read property 'previous' of null
    at Parser.parseDeclarationOrRuleset (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:192:63)
    at Parser.parseRule (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:80:21)
    at Parser.parseRulelist (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:153:27)
    at Parser.parseDeclarationOrRuleset (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:212:87)
    at Parser.parseRule (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:80:21)
    at Parser.parseRules (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:62:25)
    at Parser.parseStylesheet (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:55:47)
    at Parser.parse (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\shady-css-parser\dist\shady-css\parser.js:51:19)
    at css (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\css-slam\index.js:51:39)
    at dom5.queryAll.forEach.styleNode (C:\Users\Oneeezy\AppData\Roaming\nvm\v6.3.0\node_modules\polymer-cli\node_modules\css-slam\index.js:45:36)
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\demo\index.html_script_0.js
debug:
{ message: 'SyntaxError: Unexpected character \'#\'',
  filename: 0,
  line: 2,
  col: 10,
  pos: 11,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_0.js
debug:
{ message: 'SyntaxError: Unexpected character \'#\'',
  filename: 0,
  line: 2,
  col: 8,
  pos: 9,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_1.js
debug:
{ message: 'SyntaxError: Unexpected character \'`\'',
  filename: 0,
  line: 2,
  col: 0,
  pos: 1,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_2.js
debug:
{ message: 'SyntaxError: Unexpected character \'`\'',
  filename: 0,
  line: 2,
  col: 0,
  pos: 1,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_3.js
debug:
{ message: 'SyntaxError: Unexpected character \'`\'',
  filename: 0,
  line: 2,
  col: 0,
  pos: 1,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\marked-element\test\marked-element.html_script_4.js
debug:
{ message: 'SyntaxError: Unexpected character \'`\'',
  filename: 0,
  line: 2,
  col: 0,
  pos: 1,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at parse_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1677:9)\n    at Object.next_token [as input] (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1946:9)\n    at next (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2075:25)\n    at Object.parse (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2061:15)\n    at addFile (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:69:33)\n    at C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:77:17\n    at Array.forEach (native)\n    at Object.exports.minify (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:75:26)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\web-component-tester\data\index.html_script_0.js
debug:
{ message: 'SyntaxError: Unexpected token: operator (<)',
  filename: 0,
  line: 1,
  col: 6,
  pos: 6,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at croak (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2089:9)\n    at token_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2097:9)\n    at unexpected (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2103:9)\n    at expr_atom (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2630:9)\n    at maybe_unary (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2792:19)\n    at expr_ops (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2827:24)\n    at maybe_conditional (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2832:20)\n    at maybe_assign (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2856:20)' }
warn:    Unable to uglify file C:\Users\Oneeezy\Desktop\www_\_clients\diesel\wwwSFI\bower_components\web-component-tester\data\index.html_script_2.js
debug:
{ message: 'SyntaxError: Unexpected token: operator (<)',
  filename: 0,
  line: 2,
  col: 21,
  pos: 22,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1545:18)\n    at js_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:1553:11)\n    at croak (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2089:9)\n    at token_error (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2097:9)\n    at unexpected (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2103:9)\n    at expr_atom (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2630:9)\n    at maybe_unary (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2792:19)\n    at expr_ops (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2827:24)\n    at maybe_conditional (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2832:20)\n    at maybe_assign (eval at <anonymous> (C:\\Users\\Oneeezy\\AppData\\Roaming\\nvm\\v6.3.0\\node_modules\\polymer-cli\\node_modules\\uglify-js\\tools\\node.js:1:0), <anonymous>:2856:20)' }

```

**Comment by ziszo** [comment]
Tried: polymer build --verbose
Got no errors, just this "debug" outputs in blue.
Maybe there's nothing wrong with my files since the files that are flagged as "Unable to uglify ...." are not my files. They are mostly these marked-element.html
Maybe this warning should be ignored.

![image](https://cloud.githubusercontent.com/assets/4342864/17080908/ec03c2f4-5174-11e6-9ca6-6979b467e529.png)

![image](https://cloud.githubusercontent.com/assets/4342864/17080936/ab6045fa-5175-11e6-8c12-08a6cbc4b0a6.png)

**Comment by FredKSchott** [comment]
We've just finished work on a new build pipeline which should greatly reduce the number of files that are included in the build. This will also make sure that dependencies aren't passed through the same optimization pipeline as source files, which is where this warning is occurring. 

The good news is that these warnings shouldn't affect your builds at all, other than the fact that some files may not be minified. I'll update/close this thread once this is fixed for good with the new pipeline.

**Comment by FredKSchott** [comment]
Fixed in [v0.13.0](https://github.com/Polymer/polymer-cli/releases/tag/v0.13.0)
